scripts/r.drain/r.drain.py

```diff
@@ -141,10 +140,11 @@ def main():
     atexit.register(cleanup)
 
     if not dirmap:
+        valmap_name = valmap.split("@")[0]
         # get directions with r.fill.dir, without sink filling
-        dirmap = "%s_tmp_%d" % (valmap, os.getpid())
-        fill_map = "%s_fill_%d" % (valmap, os.getpid())
-        area_map = "%s_area_%d" % (valmap, os.getpid())
+        dirmap = grass.append_node_pid(f"{valmap_name}_tmp")
+        fill_map = grass.append_node_pid(f"{valmap_name}_fill")
+        area_map = grass.append_node_pid(f"{valmap_name}_area")
```

Comment:
Given you are changing imports anyway, can you add `import grass.script as gs` and use `gs` here for the new code (without changing the other code)? (no need to be consistent with depreciated code style)